conduct.md

```diff
@@ -11,9 +11,10 @@ redirect_from:
 ## Scala Code of Conduct
 
 We are committed to providing a friendly, safe and welcoming environment for
-all, regardless of level of experience, gender, gender identity and expression,
-sexual orientation, disability, personal appearance, body size, race, ethnicity,
-age, religion, nationality, or other such characteristics.
+all, regardless of age, body size, disability, ethnicity, sex characteristics,
+gender identity and expression, level of experience, education, socio-economic
+status, nationality, personal appearance, race, religion, or sexual identity
+and orientation.
 
```

Comment:
does this new wording come from somewhere specific, or is it original?

I'm not sure I understand the motivation for the changes, what problem is being solved. (It looks like somebody almost alphabetized the list, but not quite?)

"sex characteristics" seems like odd wording to me, I don't really know what it means. what does it mean that isn't already covered by "gender identity and expression, [...], sexual identity and orientation"?